hw/riscv/spike.c

```diff
@@ -115,7 +110,8 @@ static void create_fdt(SpikeState *s, const struct MemmapEntry *memmap,
     g_free(nodename);
 
     qemu_fdt_add_subnode(fdt, "/cpus");
-    qemu_fdt_setprop_cell(fdt, "/cpus", "timebase-frequency", 10000000);
+    qemu_fdt_setprop_cell(fdt, "/cpus", "timebase-frequency",
+        SIFIVE_CLINT_TIMEBASE_FREQ);
```

Comment:
Ah, odd -- I guess I expected Spike not to emulate a CLINT as that's kind of a SiFive thing, but it's what real spike does so it's correct to do it here.

Reply:
Yap. The CLINT models exactly the same behaviour as the old riscv_rtc.c so we eliminated the duplicate code when we added the CLINT. It's spike compatible 👍